pdk/1.0/modules/ROOT/pages/policies-pdk-apply-policies.adoc

```diff
@@ -12,19 +12,66 @@ To learn more about defining configuration parameters, see xref:policies-pdk-cre
+* If applying you policy to Flex Gateway running in Local Mode, xref:policies-pdk-publish-policies.adoc[upload your custom policy to exchange].
 
 == Connected Mode
 
 When you apply a custom policy in Connected Mode, the configuration parameters and metadata defined in `gcl.yaml` appear in the API Manager UI.
 
 To apply a custom policy in Connected Mode, see xref:gateway::policies-included-apply.adoc[Apply a Policy in Connected Mode]. 
 
+=== Update Policies in Connected Mode
+
+To update a custom policy in API Manager, you can either:
+
+* Delete the policy, and then re-apply the latest version of the policy. This automatically updates the policies binary file.
+* Update the version of the applied policy. If you update the version on the applied policy, you must check for implementation updates to ensure your policy's binary file is up to date.
++
+To check for implementation updates, see xref:exchange::manage-versions.adoc#update-policy-implementations[Update Policy Implementations].
+
 == Local Mode
 
 For Flex Gateway running in Local Mode, you apply custom policies using xref:gateway::flex-local-configuration-reference-guide.adoc[local declarative configuration files]. You specify the location of these configuration files when you first run Flex Gateway.
 
-To apply a custom policy in Local Mode, use either xref:gateway::flex-local-secure-api-with-basic-auth-policy.adoc[] or xref:gateway::flex-local-secure-api-with-auto-policy.adoc[] as an example. Replace the example configuration with your policy's configuration parameters.
+To create the local declarative files:
+
+. In the `target/wasm32-wasi/release` directory, copy both of the following files:
++
+* `<your_custom_policy>_definition.yaml`
+* `<your_custom_policy>_implementation.yaml`
++
+. Paste the two files in the `/etc/mulesoft/flex-gateway/conf.d/` Flex Gateway configuration directory.
++
+. Create a policy binding configuration file with a `.yaml` file extension to bind the policy to an API instance:
++
+* Give the file a custom name.
+* Save the file in the Flex Gateway configuration directory `/etc/mulesoft/flex-gateway/conf.d/custom`. This directory can contain multiple configuration files.
+. Copy and paste the following YAML snippet into the file, substituting your values where indicated::
++
+[source,yaml]
+----
+apiVersion: gateway.mulesoft.com/v1alpha1
+kind: PolicyBinding
+metadata:
+  name: <custom-policy-id>
+spec:
+  targetRef:
+    kind: ApiInstance
+    name: <your-api-instance>
+  policyRef:
+    kind: Extension
+    name: <your_custom_policy>-implementation
+  config:
+    <custom-cofiguration-parameters>: "your parameter"
+----
++
+* `metadata.name`: Contains the custom policy instance name.
+* `spec.targetRef.name`: Contains the name of the API instance on which to apply the custom policy. The API instance must already have been defined and applied.
+* `spec.policyRef.name`: Contains the custom policy implementation to apply.
```

Comment:
Just a suggestion, to get the policy ref name here, we could also cite the helper command we're presenting here: 
https://dev-docs-internal.kqa.msap.io/beta-pdk-ga/pdk/latest/policies-pdk-integration-tests#configure-your-custom-policy

The `make show-policy-ref-name`